concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md

```diff
@@ -1,11 +1,11 @@
+---
+##### jQuery
 
-If your data is grouped, you may want to customize the group headings. Define a [group template](/api-reference/10%20UI%20Components/dxAutocomplete/1%20Configuration/groupTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate') for that purpose. In React, implement the [groupRender](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/groupRender.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupRender') or [groupComponent](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/groupComponent.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupComponent') function.
+To customize the appearance of all Autocomplete suggestions, define an [item template](/api-reference/10%20UI%20Components/DataExpressionMixin/1%20Configuration/itemTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate'). 
 
-You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
+If your data is grouped, you may want to customize the group headings. Define a [group template](/api-reference/10%20UI%20Components/dxAutocomplete/1%20Configuration/groupTemplate.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate') for that purpose. 
 
----
-##### jQuery
+You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') if item text exceeds the drop-down list width to wrap the text.
```

Comment:
You can also enable [wrapItemText](/api-reference/10%20UI%20Components/dxDropDownList/1%20Configuration/wrapItemText.md '/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#wrapItemText') to wrap the text if the item's text exceeds the width of the drop-down list.